common/size.go

```diff
@@ -26,23 +26,23 @@ type StorageSize float64
 
 // String implements the stringer interface.
 func (s StorageSize) String() string {
-	if s > 1000000 {
-		return fmt.Sprintf("%.2f mB", s/1000000)
-	} else if s > 1000 {
-		return fmt.Sprintf("%.2f kB", s/1000)
+	if s > 1048576 {
+		return fmt.Sprintf("%.2f mb", s/1048576)
```

Comment:
Is this really correct? The term `mb` means `1024 * 1024` bits, not bytes. IIUC:
* lowercase `b` is `bit` and uppercase `B` is `byte`
* lowercase `m` is `1024*1024`, and uppercase `M` is `1000000`

Reply:
It gets even more messy if you introduce the concept of mebibytes :D https://en.wikipedia.org/wiki/Megabyte

All in all I think in software development everyone is using the byte notation for everything, and everyone is calculating using 1024 for everything. The lowercase conversion was just to make is consistent in the logs and not have storage slots uppercase and time values lowercase.

Reply:
It's your call, but I prefer to use the correct term. So currently it looks like you should use `MiB`, or revert to how it was calculated before, but use the term `MB`. 
If you're dead-set on lowercase, and want to wage war against ISO standards, you could use `mib`.

Reply:
Ok, I'll rename to MiB